docs/localization/LOCALIZATION.md

```diff
@@ -74,343 +78,443 @@ export function localizationConfigurationFactory() {
+    // ...
+  }
 
-  public getMyPresContext(overrideContext) {
-    return {
-      translations,
-      inputs: {
-        // ...
-      }
-    };
+  /**
+   * Called upon language change to set current language
+   * @param  language
+   */
+  public useLanguage(language: string) {
+    this.localizationService.useLanguage(language);
   }
 }
 ```
 
-A container can have its own translation (for error messages for example). In this case the container become `Translatable` as following:
+Now we can start using pipes:
+
+```html
+<p>{{today | date: 'fullDate' : '' : currentLanguage}}</p>
+
+<p>{{1.5487 | number: '' : currentLanguage}}</p>
+
+<p>{{1.5487 | currency: 'EUR' : 'symbol' : '' : currentLanguage}}</p>
+```
+
+### How to add custom translations in the container
+
+A container can have its own translations (for error messages for example). In this case, the container becomes `Translatable` as follows:
 
 ```typescript
-import {MyPresTranslation} from '../presenter/my.translation.ts';
-import {MyContTranslation} from '../my.translation.ts';
-import {Translatable} from '@o3r/localization';
+import { Translatable } from '@o3r/localization';
+import { MyContTranslation } from '../my.translation.ts';
 
 @Component({
   selector: 'o3r-my-cont',
   templateUrl: './my.template.html',
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class MyContComponent implements FormsPocContContext, Translatable<MyContTranslation>, Block {
+export class MyContComponent implements Translatable<MyContTranslation>, Block {
 
   /**
    * Localization of the component
    */
   @Input()
   @Localization('./my.localization.json')
-  public translations: Partial<MyPresTranslation> & MyContTranslation;
-
-  ...
-
-  public getMyPresContext(overrideContext) {
-    return {
-      translations,
-      inputs: {
-        // ...
-      }
-    };
-  }
+  public translations: MyContTranslation;
 }
 ```
 
-### Configure TranslateService in your root component app.component.ts
+There is also a possibility for the container to override the presenter's translations and pass them as input to the presenter.
 
-```typescript
+### Configure the Localization Service in your root component
 
+```typescript
 // app.component.ts
 
-import { LocalizationService } from  "@o3r/localization";
-
-
+import { LocalizationService } from '@o3r/localization';
 
 // Inject LocalizationService which will take care of configuring TranslateService using LocalizationConfiguration and call configure() method
-
-constructor(private router: Router,
-  private localizationService: LocalizationService) {
-    this.localizationService.configure();
+constructor(private localizationService: LocalizationService) {
+  this.localizationService.configure();
 }
-
 ```
 
 ### How to add RTL support in my app
 
-The **TextDirectionService** has to be injected in `app.component.ts` as follows.
+The `TextDirectionService` has to be injected in `app.component.ts` as follows:
```

Comment:
To be discussed with the team how we can move it outside the app.component to not pollute it